test/boost/sstable_resharding_test.cc

```diff
@@ -30,8 +30,7 @@ static inline std::vector<std::pair<sstring, dht::token>> token_generation_for_s
     std::vector<std::pair<sstring, dht::token>> key_and_token_pair;
 
     key_and_token_pair.reserve(tokens_to_generate);
-    dht::set_global_partitioner(to_sstring("org.apache.cassandra.dht.Murmur3Partitioner"));
-    dht::murmur3_partitioner partitioner(smp::count, 0);
+    dht::murmur3_partitioner partitioner(smp::count, 12);
 
```

Comment:
This is more than "sstable_resharding_test: stop calling set_global_partitioner".

Reply:
changed.